firstTimeInstall.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+locale  # check for UTF-8
+
+set -e
+
+sudo apt update && sudo apt install -y locales
+sudo locale-gen en_US en_US.UTF-8
+sudo update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+export LANG=en_US.UTF-8
+
+locale  # verify settings
+
+sudo apt install -y software-properties-common
+sudo add-apt-repository -y universe
+
+sudo apt update && sudo apt install curl -y
+sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg
+
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | sudo tee /etc/apt/sources.list.d/ros2.list > /dev/null
+
+sudo apt update
+
+sudo apt upgrade -y
+
+sudo apt install -y \
+ros-humble-desktop \
+ros-humble-ros-base \
```

Comment:
`ros-humble-desktop` includes ros-humble-ros-base, you can get rid of ros-humble-ros-base